app/views/camps/_list.html.erb

```diff
@@ -40,6 +40,10 @@
 
                             <% end %>
                         </div>
+
+                        <% if !camp.is_public? %>
+                            <h4 style="color: red; position: absolute; top:0; right:0;">לא פומבי</h4>
```

Comment:
Let's use he.yml for the ״לא פומבי״ and also if you're there please add it to en.yml as in the future we want other communities to use this framework